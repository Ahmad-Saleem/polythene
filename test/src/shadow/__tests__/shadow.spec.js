/* global describe */
import {runTests} from '../../../scripts/testing';
import {tests} from '../';

describe('Shadow component', () => {
    runTests(tests);
});