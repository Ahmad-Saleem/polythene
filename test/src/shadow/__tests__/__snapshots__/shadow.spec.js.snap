exports[`Shadow component appearance params 1`] = `
"<div id=\"\" class=\"pe-shadow pe-shadow--animated\">
  <div class=\"pe-shadow__bottom pe-shadow--z-5\"></div>
  <div class=\"pe-shadow__top pe-shadow--z-5\"></div>
</div>
"
`;

exports[`Shadow component common component params 1`] = `
"<div id=\"\" class=\"pe-shadow\">
  <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
  <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
</div>
"
`;

exports[`Shadow component no params 1`] = `
"<div id=\"\" class=\"pe-shadow\">
  <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
  <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
</div>
"
`;

exports[`Shadow component param content 1`] = `
"<div id=\"\" class=\"pe-shadow\">
  <div>
    CONTENT
  </div>
  <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
  <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
</div>
"
`;
